### PR TITLE
Update demo docs

### LINKED ISSUE #185: [DOC] 提供Non root相关的使用文档
**Provide a link to that doc page:**

**What is the defect and your suggestions on improvement:**

### PR DESCRIPTION
**Pull request by TrafalgarZZZ**
### Ⅰ. Describe what this PR does
Update demo docs:
- Update demo doc "accelerate data accessing"
- Update demo doc "data co-locality"
- Add demo doc "guide for non-root users"
- Update related samples
- Update TOC

### Ⅱ. Does this pull request fix one issue?

fixes #185 

### Ⅴ. Special notes for reviews
`hostpath.md` should also be updated. Working on it

### CODE REVIEW

File: docs/zh/samples/nonroot_access.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# 示例 - 以non-root用户身份使用Fluid
```
Comment (cheyang) [inline docs/zh/samples/nonroot_access.md:1]: 是不是应该改成 使用Fluid访问非root用户的数据

File: docs/zh/samples/nonroot_access.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+在使用Fluid时，用户不必拥有root权限即可创建Dataset，将自己的数据集挂载到Fluid上，并创建相关应用消费这些数据
```
Comment (cheyang) [inline docs/zh/samples/nonroot_access.md:3]: 如果用户的数据只能以特定uid访问时，需要通过设置Runtime的RunAs参数指定特定用户运行Cache Engine访问底层数据。

**Review by cheyang — CHANGES_REQUESTED** [review]

File: docs/zh/samples/nonroot_access.md (hunk 2 above)
Comment (TrafalgarZZZ) [reply docs/zh/samples/nonroot_access.md]: 感谢建议 已修改

File: docs/zh/samples/nonroot_access.md (hunk 1 above)
Comment (TrafalgarZZZ) [reply docs/zh/samples/nonroot_access.md]: 已修改

**Pull request merged**